pkg/ccl/changefeedccl/sink_kafka_v2.go

```diff
@@ -301,7 +309,11 @@ type kafkaBuffer struct {
 	batchCfg sinkBatchConfig
 }
 
-func (b *kafkaBuffer) Append(key []byte, value []byte, attrs attributes) {
+type mvccTS string
+
+const mvccTSKey mvccTS = "mvccts"
```

Comment:
to make it more type safe, you should make the key type a private struct type

e.g. something like
```
type mvccTSKey struct{}
...
context.WithValue(ctx, mvccTSKey{}, ...)
```

Reply:
ah good point. my comment still stands from an idiomatic point of view 😛 

see the doc for https://pkg.go.dev/context#WithValue:
> To avoid allocating when assigning to an interface{}, context keys often have concrete type struct{}.